.github/workflows/cli.yml

```diff
@@ -0,0 +1,56 @@
+name: CLI weekly live test
+
+on:
+  schedule:
+    - cron: "0 18 * * 0"
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  apic-extension-live-test:
+    runs-on: ubuntu-latest
+    environment: engineering
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.11"
+
+      - name: Azure login
+        uses: azure/login@v2
+        with:
+          client-id: ${{secrets.APICEXT_TEST_AZURE_CLIENT_ID}}
+          tenant-id: ${{secrets.APICEXT_TEST_AZURE_TENANT_ID}}
+          subscription-id: ${{secrets.APICEXT_TEST_AZURE_SUBSCRIPTION_ID}}
+          enable-AzPSSession: true
+
+      - name: Clone azure-cli-extensions repository
+        run: |
+          git clone https://github.com/blackchoey/azure-cli-extensions.git
+          cd azure-cli-extensions
+          git checkout dev
+
+      - name: Setup development environment
+        run: |
+          cd azure-cli-extensions
+          python -m venv .venv
+          source .venv/bin/activate
+          python -m pip install -U pip
+          pip install azdev
+          azdev setup -r .
+          azdev extension add apic-extension
+
+      - name: Set environment variable
+        run: |
+          echo "USERASSIGNED_IDENTITY=${{ secrets.USERASSIGNED_IDENTITY }}" >> $GITHUB_ENV
```

Comment:
Since you set env in line 56, is this step still needed?

Reply:
removed